CITATION.bib

```diff
@@ -30,13 +30,12 @@ @article{schlottkelakemper2021purely
   eprintclass={math.NA}
 }
 
-@misc{schlottkelakemper2020trixi,
+@misc{schlottkelakemper2025trixi,
   title={{T}rixi.jl: {A}daptive high-order numerical simulations
          of hyperbolic {PDE}s in {J}ulia},
   author={Schlottke-Lakemper, Michael and Gassner, Gregor J and
-          Ranocha, Hendrik and Winters, Andrew R},
-  year={2020},
-  month={08},
+          Ranocha, Hendrik and Winters, Andrew R and Rueda-Ramírez, Andrés},
```

Comment:
```suggestion
          Ranocha, Hendrik and Winters, Andrew R and
          Chan, Jesse and Rueda-Ramírez, Andrés},
```
Historically, we have always appended to this list. For now, I suggest we continue with this procedure (but I'm open to discussing it for the future).